docker-compose.yml

```diff
@@ -827,6 +827,22 @@ services:
         max-size: "${DEFAULT_LOG_SIZE}"
         max-file: "${HIGH_PRIO_LOG_FILES}"
 
+  #
+  # FAF Solr standalone server
+  #
+  faf-solr:
+    # version: '3'
+    container_name: faf-solr
+    image: solr:8
+    ports:
+      - "8983:8983"
```

Comment:
I don't think we want to expose this part to the world? The API should use it internally.
Instead we need a network alias called faf-solr (see other services as example)